src/service/storage-impl.js

```diff
@@ -181,7 +178,7 @@ export class Store {
    */
   constructor(obj, opt_maxValues) {
     /** @const {!JSONType} */
-    this.obj = recreateNonProtoObject(obj);
+    this.obj = /** @type {!JSONType} */ (recreateNonProtoObject(obj));
```

Comment:
I got a type error that's why I casted this. Let me know if this should be something else.